src/taskpane/index.tsx

```diff
@@ -7,21 +7,21 @@ import { FluentProvider, webLightTheme } from "@fluentui/react-components";
 
 const title = "Contoso Task Pane Add-in";
 
-const rootElement: HTMLElement = document.getElementById("container");
-const root = createRoot(rootElement);
+const rootElement: HTMLElement | null = document.getElementById("container");
```

Comment:
This was another thing noticed due to turning strict on. document.getElementById() can return null, and even though Intellisense says it only returns HTMLElement, it's really `HTMLElement | null`. This allows the root element to be null, but effectively does nothing when that is the case using `root?.render()` .